src/server/lib/templateRules.json

```diff
@@ -5503,7 +5612,7 @@
       "key": "Is_project_designed_to_meet_100_mbps__c",
       "index": 33,
       "required": "Conditional",
-      "dataType": "TBD",
+      "dataType": "Pick List",
```

Comment:
this has become a `Pick List` but the `listVals` still read `TBD`, meaning that will be the only value permitted by render. we should source the pick list values now @jcomeau1

Reply:
Maybe this is a yes/no ?

Reply:
It is a Yes/No. I wonder if you have the ability to see the 'Data Validation' of a field? If so, it would give you the appropriate reference within the Dropdowns sheet.

Separately, I have replaced TBD with the values. I'll proved the revision once I hear back from the partners on the above questions.

Reply:
I added the metadata to the slightly revised verion I just sent. I would attach it, but unfortunately GitHub does not allow this. Check your email or Google Drive.